app/src/test/java/com/codesoom/assignment/controllers/TaskControllerTest.java

```diff
@@ -0,0 +1,132 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskController 를 테스트 한다.")
+class TaskControllerTest {
+    private static final String TASK_TITLE_1 = "title1";
+    private static final String TASK_TITLE_2 = "title2";
+    private Task task1;
+    private Task task2;
+    private List<Long> invalidIds;
```

Comment:
주어진 조건이라는 사실을 표현하기 위해 이름을 수정하는 것도 좋겠습니다.
```suggestion
    private List<Long> givenInvalidIds;
```